lib/hooks/userspace/hook-express.js

```diff
@@ -101,12 +103,8 @@ function endRootSpanForRequest(rootContext, req, res) {
 }
 
 module.exports = {
-  patch: function(express_, agent_) {
-    // We only support express v4. In the past express exported a version property
-    // that would have been great for this purpose, but v3.x removed it. Instead
-    // we rely on 'createServer' export that was removed in v4.
-    // See: https://github.com/strongloop/express/wiki/Migrating-from-3.x-to-4.x
-    if (typeof module.createServer !== 'undefined') {
+  patch: function(express_, agent_, version_) {
+    if (SUPPORTED_VERSIONS.indexOf(version_.split('.')[0]) === -1) {
```

Comment:
Use the [semver](https://www.npmjs.com/package/semver) module instead of hand-rolling this.